requirements.txt

```diff
@@ -1,8 +1,8 @@
 # generated from manifests external_dependencies
-cryptography
-jira
-oauthlib
-PyJWT
-requests
-requests-oauthlib
-requests-toolbelt
+cryptography==2.6.1
```

Comment:
@leemannd You should not do that.

See https://github.com/OCA/connector-jira/blob/15.0/.pre-commit-config.yaml#L120

If you have some limitations due to some packages versions, add it to external_dependencies key in your module manifest. This will be automatically updated by pre-commit.

Note: Prefer using >= in external_dependencies key than ==

Reply:
Hello, usually don't update this. I wasn't sure about the change. Thanks for the input